app/actions/index.js

```diff
@@ -125,7 +168,7 @@ export const clearRemoteSurveys = () => (dispatch, getState) =>
     type: types.CLEAR_REMOTE_SURVEYS
   });
 
-export const fetchRemoteSurvey = (id, url) => (dispatch, getState) => {
+export const fetchRemoteSurvey = (id, url, target) => (dispatch, getState) => {
```

Comment:
No, that all makes sense. The mismatch is that survey definitions aren't 1:1 with OSM data to be sync'd between devices. There may be multiple surveys available (and fetched) but only 1 sync-worthy OSM data instance.

If someone fetches a survey, it will sync the OSM data. If they then sync _another_ survey, it will sync the OSM data **again**.